_events/2022/2022-11-sc22.md

```diff
@@ -14,17 +14,15 @@ time:
 We are excited to announce the Research Software Engineers in HPC Workshop (RSE-HPC-2022)
 to be held as part of [SC22](https://sc22.supercomputing.org/) (hybrid event, in Dallas, TX and virtually).
 This will be a half-day workshop 2:00pm-5:30pm CST (UTC-6), Sunday, November 13, 2022.
+The theme for this year’s workshop is "How Do RSEs and HPC Centers Fit Together?"
 
 This workshop will bring together RSEs and allies involved in HPC, from all over the world,
 to grow the RSE community by establishing and strengthening professional networks of current
 RSEs and RSE leaders.  We’ll discuss the current activities and plans of national RSE
 organizations, discuss the needs of RSEs and RSE groups, and brainstorm ways RSE organizations
 can help address these.
 
-We encourage prospective participants to submit position papers (limit of 2 pages,
-not counting references; no format prescribed) on topics related to RSE issues.
-Papers and discussion topics should be submitted using the [submission website](https://submissions.supercomputing.org/?page=Submit&id=SC22WorkshopRSEHPC2022Submission&site=sc22).
-The deadline for submissions is Friday, August 5, 2022, at 11:59 PM AoE (UTC-12).
+Participation is open to all SC22 attendees who have signed up for the “Workshops” option.
```

Comment:
maybe: "Participation is open to all SC22 attendees who have registered to attend the conference workshops."?

Reply:
Sounds fine, will do.